test/parallel_api/ranges/std_ranges_stable_sort.pass.cpp

```diff
@@ -24,10 +24,8 @@ main()
 
     auto sort_stable_checker = TEST_PREPARE_CALLABLE(std::ranges::stable_sort);
 
-    test_range_algo<0, int, data_in, DevicePolicy>{big_sz}(dpl_ranges::stable_sort, sort_stable_checker);
-
-    test_range_algo<1, int, data_in, DeviceAndParPolicies>{medium_sz}(dpl_ranges::stable_sort, sort_stable_checker,
-                                                                      std::ranges::less{});
+    test_range_algo<0>{big_sz}(dpl_ranges::stable_sort, sort_stable_checker);
```

Comment:
Did you mean to remove this `big_sz` here?

Reply:
Yep, removed